kubernetes/schema_job_spec.go

```diff
@@ -36,14 +36,12 @@ func jobSpecFields() map[string]*schema.Schema {
 		"active_deadline_seconds": {
 			Type:         schema.TypeInt,
 			Optional:     true,
-			ForceNew:     true,
```

Comment:
This used to force recreation of the resource so the previous acceptance passed anyway: https://github.com/terraform-providers/terraform-provider-kubernetes/pull/769/files#diff-de5bc44a1e838d53cd603bc71a631b48R74